gk/main.c

```diff
@@ -2271,7 +2254,26 @@ gk_proc(void *arg)
 			 * check if it's expired.
 			 */
 			rte_prefetch_non_temporal(fe);
-		}
+
+			iter_count = 0;
+		} else
+			iter_count++;
+
+		process_pkts_front(port_front, port_back,
+			rx_queue_front, tx_queue_back,
+			lcore, &front_num_pkts, front_icmp_bufs,
+			instance, gk_conf);
+
+		process_pkts_back(port_back, port_front,
+			rx_queue_back, tx_queue_front, lcore,
+			&back_num_pkts, back_icmp_bufs, instance, gk_conf);
+
+		if (fe != NULL && fe->in_use && is_flow_expired(fe, rte_rdtsc())) {
+			flow_hash_val = rss_ip_flow_hf(&fe->flow, 0, 0);
+			rte_hash_prefetch_buckets(instance->ip_flow_hash_table,
```

Comment:
Use the non-temporal version of `rte_hash_prefetch_buckets()`.

Reply:
Done.